app/views/lists/by_login.html.erb

```diff
@@ -60,7 +64,7 @@
     </div>
   <% end %>
 <% end %>
-<%= render :partial => "shared/by_login_header", :locals => { :after => tools } %>
+<%= render :partial => "shared/by_login_header", :locals => { :after => tools, title: observation_by_login_title } %>
```

Comment:
James asked for a custom title in one specific page, so I introduced a new translation string along with a default. Not incredibly pretty, but I think it's not too bad.